client/src/components/BackToTop/BackToTop.stories.js

```diff
@@ -4,6 +4,11 @@ import React, { Fragment } from "react";
 
 import { BackToTop } from "./BackToTop.js";
 
+function focusBTT () {
```

Comment:
we generally name our variables using `underscore_notation`. So please excuse the functions where it follows camel casing (i need to go back and fix those).
With that said, generally we use camel notation when we're accepting props (but I've only noticed us using camel case notation either when the library we're using uses it, or if I do it out of habit and I forget to go back and fix it

But in this case, I don't think there's much efficiency to be gained by setting a function in memory so we don't really need to rename this function. With that, we can create an anonymous arrow function

```
focus={() => {
  //code here
}}
```

(If you could rename the bottom one i'd appreciate it!)

Reply:
Forgot to mention that there's not much efficiency to be gained because everything gets re-rendered when the code changes (in storybook) anyways which means that the memory would've been reset anyways

(I think that's how storybook works anyways)